libweb3jsonrpc/Test.cpp

```diff
@@ -23,16 +23,52 @@
 #include "Test.h"
 #include <jsonrpccpp/common/errors.h>
 #include <jsonrpccpp/common/exception.h>
-#include <libethereum/ClientTest.h>
+#include <libdevcore/CommonJS.h>
 #include <libethereum/ChainParams.h>
+#include <libethereum/ClientTest.h>
 
 using namespace std;
 using namespace dev;
+using namespace dev::eth;
 using namespace dev::rpc;
 using namespace jsonrpc;
 
 Test::Test(eth::Client& _eth): m_eth(_eth) {}
 
+namespace
+{
+string logEntriesToLogHash(eth::LogEntries const& _logs)
+{
+    RLPStream s;
+    s.appendList(_logs.size());
+    for (eth::LogEntry const& l : _logs)
+        l.streamRLP(s);
+    return toHexPrefixed(sha3(s.out()));
```

Comment:
It would be the same result, but for consistency I'd use either `toHexPrefixed` or `toJS` in both places - here and in `return toJS(dev::EmptyListSHA3);`